### PR TITLE
design project migration

### PR DESCRIPTION
**Pull request by TDM1911**
## Related Tickets:
- [#29919](https://edu-redmine.sun-asterisk.vn/issues/29919)
- [#29935](https://edu-redmine.sun-asterisk.vn/issues/29935)
- [#29953](https://edu-redmine.sun-asterisk.vn/issues/29953)

## WHAT this PR does?
- Add project's table migration.

## Notes Impacted Areas:
- N/A.

## Attachments:
ảnh

### CODE REVIEW

**Comment by TDM1911** [comment]
ready

File: database/migrations/2020_09_29_030333_create_attachements_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class CreateAttachementsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('attachements', function (Blueprint $table) {
```
Comment (huyenttm-0901) [inline database/migrations/2020_09_29_030333_create_attachements_table.php]:
```suggestion
        Schema::create('attachments', function (Blueprint $table) {
```

File: database/migrations/2020_09_29_030333_create_attachements_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('attachements');
```
Comment (huyenttm-0901) [inline database/migrations/2020_09_29_030333_create_attachements_table.php]: sửa nhé

**Comment by TDM1911** [comment]
ready

**Pull request merged**